detect/region.go

```diff
@@ -0,0 +1,66 @@
+package detect
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/urfave/cli"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/utils"
+
+	"github.com/Scalingo/go-scalingo/v4/debug"
+)
+
+// CurrentRegion extract and returns the region from the Git remote URL
+// If not found it return an empty string
+func CurrentRegion(c *cli.Context, rc *config.RegionsCache) string {
```

Comment:
I don't find the function name really good. Moreover I'm not sure it adds anything to the `getRegionFromGitRemote` function. Maybe it should be merged into the `getRegionFromGitRemote` function?

Reply:
function was named this way to stay consistent with the app detection function. In the app part, this function makes sense but it's not really relevant here. I will merge this function into `getRegionFromGitRemote`